test/rails_app/app/views/free_invitations/new.html.erb

```diff
@@ -1,12 +1,16 @@
 <h2>Send invitation</h2>
 
-<%= form_for resource, :as => resource_name, :url => free_invitation_path do |f| %>
+<%= form_for(resource, as: resource_name, url: free_invitation_path) do |f| %>
   <%= devise_error_messages! %>
```

Comment:
I think it should be changed to render partial

Reply:
Done! See latest commit (https://github.com/scambra/devise_invitable/pull/786/commits/568ea18b66e879fe3c8ef72ab04ce6981833f61d)